.github/workflows/velocity.yml

```diff
@@ -68,8 +63,10 @@ jobs:
         curl \
           --data-urlencode "format=markdown" \
           --data-urlencode "title=Report for ${REPORT_DATE}" \
-          --data content="$(cat report.md)" \
-          --silent --fail --show-error \
+          --data-urlencode content@report.md \
```

Comment:
If possible, I'd prefer `--form` because it doesn't need to urlencode special characters in the markdown